distributed/deploy/tests/test_local.py

```diff
@@ -41,6 +42,13 @@ def test_simple(loop):
             assert e.loop is c.loop
 
 
+def test_local_cluster_supports_blocked_handlers(loop):
+    with LocalCluster(blocked_handlers=['run_function'], loop=loop) as c:
+        with Client(c) as client:
+            with pytest.raises(CommClosedError):
+                client.run_on_scheduler(lambda x: x, 42)
```

Comment:
Yeah, so we shouldn't have a closed comm, we should get the `ValueError`.  This is related to the comment above about handling errors in core.py

Reply:
updated